build.sh

```diff
@@ -57,6 +57,24 @@ do
   esac
 done
 
+if [ "$DO_UPDATE" == true ] && [ "$QUIET" == false ]; then
+  echo "Checking if html-build is up to date..."
+  ORIGIN_URL=$(git remote get-url origin)
+  git fetch -q $ORIGIN_URL master
```

Comment:
Even if there are new commits? Kind of a shame you don't get to see how behind you are.

But yeah, OK, sounds good to me.